README.md

```diff
@@ -162,7 +164,9 @@ The gRPC service can be used to start a server which is able to perform back-to-
 
 ```bash
 ➜ bazel-bin/nighthawk_service --help
```

Comment:
I noticed that the -> is used here but not above in the "Test it" and "Build it" sections.   Is that intentional?

Reply:
@oschaaf

Reply:
No the inconsistency is not intentional. I have been manually updating these sections, but actually I consider the source of truth for this to live over at  https://github.com/envoyproxy/nighthawk/blob/master/source/client/options_impl.cc
I wonder if it's possible to somehow automate updating these docs from that.